front/app/components/admin/ContentBuilder/Toolbox/Container.tsx

```diff
@@ -15,7 +15,7 @@ const Container = ({ children }: Props) => (
     position="fixed"
     zIndex="99999"
     flex="0 0 auto"
-    h="100%"
+    h={`calc(100vh - ${stylingConsts.menuHeight}px)`}
```

Comment:
Did you check that this doesn't affect the report builder?

Reply:
It does, thanks! I'll fix it.